k9mail/src/main/res/layout-land/accounts.xml

```diff
@@ -1,115 +1,127 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/root_view"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+    android:layout_height="match_parent">
 
-    <include layout="@layout/simple_toolbar_layout" />
-<LinearLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="horizontal">
+    <include
+        android:id="@+id/appbar"
+        layout="@layout/simple_toolbar_layout"
+        app:layout_constraintTop_toTopOf="parent " />
 
-    <LinearLayout
-        android:layout_width="wrap_content"
+    <TextView
+        android:id="@+id/global_settings_title"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:orientation="vertical">
-
-        <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:layout_marginBottom="8dp"
-            android:fontFamily="sans-serif-medium"
-            android:paddingStart="16dp"
-            android:paddingEnd="16dp"
-            android:text="@string/global_settings_title"
-            android:textColor="?attr/colorAccent" />
-
-        <FrameLayout
-            android:id="@+id/generalSettingsContainer"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            tools:layout_height="300dp" />
+        android:layout_marginTop="16dp"
+        android:layout_marginBottom="8dp"
```

Comment:
what is the purpose of 8dp, is that aligned with the Material design guidelines?

Reply:
I am just keeping what we had before there, just refactored the layout but did not change any sizes etc.